src/components/swap/components/select-token-modal.tsx

```diff
@@ -101,42 +101,49 @@ const TokenItem = ({
   isDarkMode,
   item,
   onClick,
-}: TokenItemProps) => (
-  <Flex
-    align='center'
-    justify='space-between'
-    cursor='pointer'
-    h='60px'
-    my='4px'
-    px='16px'
-    onClick={() => onClick(item.symbol)}
-    _hover={{
-      backgroundColor: isDarkMode ? colors.gray[900] : colors.gray[100],
-    }}
-  >
-    <Flex align='center'>
-      <Image alt={`${item.symbol} logo`} src={item.image} w='40px' h='40px' />
-      <Flex direction='column' ml='16px'>
-        <Flex align='baseline'>
-          <Text fontSize='md' fontWeight='500' textColor={colors.icBlack}>
-            {item.symbol}
-          </Text>
-          {extraTitle && (
-            <Text color={colors.icBlue} fontSize='sm' fontWeight='500' ml='2'>
-              {extraTitle}
+}: TokenItemProps) => {
+  return (
+    <Flex
+      align='center'
+      justify='space-between'
+      cursor='pointer'
+      h='60px'
+      my='4px'
+      px='16px'
+      onClick={() => onClick(item.symbol)}
+      _hover={{
+        backgroundColor: isDarkMode ? colors.ic.gray[900] : colors.ic.gray[100],
+      }}
+    >
+      <Flex align='center'>
+        <Image alt={`${item.symbol} logo`} src={item.image} w='40px' h='40px' />
+        <Flex direction='column' ml='16px'>
+          <Flex align='baseline'>
+            <Text fontSize='md' fontWeight='500' textColor={colors.ic.black}>
+              {item.symbol}
             </Text>
-          )}
+            {extraTitle && (
+              <Text
+                className='text-ic-blue-500'
+                fontSize='sm'
+                fontWeight='500'
+                ml='2'
+              >
+                {extraTitle}
+              </Text>
+            )}
+          </Flex>
+          <Text fontSize='sm' fontWeight='500' textColor={colors.ic.black}>
+            {item.name}
+          </Text>
         </Flex>
-        <Text fontSize='sm' fontWeight='500' textColor={colors.icBlack}>
-          {item.name}
+      </Flex>
+      <Flex align='flex-end' direction='column' ml='16px'>
+        <Text fontSize='md' fontWeight='700' textColor={colors.ic.black}>
+          {balance}
         </Text>
+        <Text fontSize='sm' fontWeight='500'></Text>
```

Comment:
Is this used for anything visually, or can remove?

Reply:
Removed.